ferrocene/doc/internal-procedures/src/release/stable.rst

```diff
@@ -59,10 +59,61 @@ the latest commit on that branch into the ``beta-${major_version}`` channel
 every night. You can continue landing changes into the branch until you
 are ready to release it as a stable release.
 
+Add version to Known Problems
+-----------------------------
+
+Add the new version to the Known Problems database by adding the version and branch
+to the ``src/config.yml`` file then running the tool.
```

Comment:
maybe point to /problems repo, because otherwise it looks like these sorts of problems live in this one